airbyte_cdk/test/standard_tests/_job_runner.py

```diff
@@ -21,26 +21,6 @@
 )
 
 
-def _errors_to_str(
```

Comment:
This functionality has moved in the `EntrypointOutput` class so its available to all instances of `EnterpointOutput`.